### DISCUSSION TITLE
3.0.0 alfred.app onboarding

### DISCUSSION THREAD

**Discussion by zhongjis** [blacs30/bitwarden-alfred-workflow#159] [Ideas]
This is a general topic served as a communication channel regarding the topics of alfred.app integration.
You can see the Git project [here](https://github.com/users/zhongjis/projects/1/views/1)

Let me know if you have any questions or suggestions :)

**Comment by zhongjis** [comment]
env var migration completed. https://github.com/blacs30/bitwarden-alfred-workflow/pull/158

**Reply by blacs30** [reply]
Nice, great job. 
If you/we want to release a new version we need to create a new tag and push it to the repo, then the release build starts but I think that can still wait after all the changes have been made.

**Comment by zhongjis** [comment]
> Nice, great job. If you/we want to release a new version we need to create a new tag and push it to the repo, then the release build starts but I think that can still wait after all the changes have been made.

agreed. lets wait until all things are merged

**Reply by blacs30** [reply]
@zhongjis I just released the 3.0.0 with the changes you made and an additional fix to work again with the latest bitwarden cli. Sorry it took me so long. It's a nice way to configure the workflow via the ui. I think a change might be needed in the code to set the server url. I noticed that just updating the var/config doesn't set it in`bw config server`.

**Reply by zhongjis** [reply]
no worries. i caught up with my work a bit recently and havent got a chance to sign the binaries yet. will post here once I continue my work on that. if you have got time, feel free to do so and after that I think we can post the workflow on the store.

meanwhile my workflow is broken locally which I think relates to the server url thingy. will post here again if anything new I found

**Comment by zhongjis** [comment]
more info on alfred.app release, see #187
